src/data/monsters.txt

```diff
@@ -583,6 +583,18 @@ serialbus	982	serialbus.gif	Atk: 60 Def: 54 HP: 60 Init: 50 Meat: 30 P: demon E:
 Servant of Grodstank	799	grodseal.gif	NOCOPY FREE Atk: 175 Def: 157 HP: 200 Init: -10000 P: horror EA: stench Article: a	fustulent seal grulch (35)	infernal toilet brush (5)
 sewer snake with a sewer snake in it	1752	sewersnake.gif	Atk: 2 Def: 1 HP: 2 Init: -10000 P: beast SNAKE E: stench Article: a	carton of snake milk (100)	carton of snake milk (30)	sewer snake (30)
 shadow of Black Bubbles	798	shadowseal.gif	NOCOPY FREE Atk: 175 Def: 157 HP: 200 Init: -10000 P: horror EA: spooky Article: a	scrap of shadow (35)	shadowy seal eye (5)
+shadow bat	2271	shadowbat.gif	Atk: 100 Def: 100 HP: 100 Init: 300 P: horror Article: a	shadow sausage (5)
```

Comment:
all these guys are `EA: cold EA: spooky`. prism is also `EA: hot`.

Is the drop rate being 5% for every monster a dev leak? If not good job getting all 12 within the day!

Reply:
It's not a dev leak, it was just the current theory. Figured it was better than nothing - would you rather a (0)? What normally means unspaded here?

Reply:
yeah (0) means unspaded.

It probably is better than nothing but I'm curious as to what it's based on. Did somebody PFC one of the drops on a copied monster? Turns divided by items? Ran autumn-aton on the rift with one acuity, no drops, two acuity, some drops? Just eyeballing, eh, seems reasonable?